### PR TITLE
Add traefik image using non-root

### PR DESCRIPTION
**Pull request by drpaneas**

### CODE REVIEW

File: traefik-image/traefik-image.kiwi.ini [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        <entrypoint execute="/entrypoint.sh"/>
+        <subcommand clear="true"/>
+        <expose>
+          <port number="80"/>
+        </expose>
+        <volumes>
+          <volume name="/tmp"/>
+        </volumes>
+        <labels>
+           <label name="org.label-schema.description" value="A modern reverse-proxy"/>
```
Comment (davidcassany) [inline traefik-image/traefik-image.kiwi.ini]: Are those labels mandatory? I think we should follow the https://fate.suse.com/326856 the base images just started adopting so and it is still to be seen how we want to add them in CaaSP/Kubic. If not mandatory I would just leave them without those labels and just add them for all images at once following a clear criteria.

There is a card on the backlog regarding that in [v4](https://trello.com/c/rp3dq9AV/44-add-labels-schema-to-our-caasp-images-for-sle15)

Comment (drpaneas) [reply traefik-image/traefik-image.kiwi.ini]: I don't think they are needed -- I've removed it.

File: traefik-image/traefik-image.kiwi.ini [hunk 2]
```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- OBS-AddTag: _NAMESPACE_/traefik:%%LONG_VERSION%% _NAMESPACE_/traefik:%%LONG_VERSION%%-<RELEASE> -->
+
+<image schemaversion="6.5" name="_PRODUCT_-traefik-image">
+  <description type="system">
+    <author>SUSE Containers Team</author>
+    <contact>containers@suse.com</contact>
+    <specification>nginx ingress controller running on a _DISTRO_ container guest</specification>
```
Comment (davidcassany) [inline traefik-image/traefik-image.kiwi.ini]: Sorry missed that on the first review, but the description needs to be updated :)

Comment (drpaneas) [reply traefik-image/traefik-image.kiwi.ini]: sorry I missed that. thanks for the spotting it ;)

**Pull request merged**